pangeo-notebook/packages.txt

```diff
@@ -106,26 +106,27 @@ contourpy==1.0.7
 coolname==2.2.0
 croniter==1.3.8
 cryptography==39.0.0
+cudatoolkit==11.8.0
```

Comment:
hmmm, not sure where this is coming from, but would prefer to keep cudatoolkit just in ML images...

Reply:
pyarrow apparently...
```
cudatoolkit==11.8.0
  └─ ucx 1.12.1 [required: >=11.2,<12]
     └─ libarrow 11.0.0 [required: >=1.12.1,<1.13.0a0]
        ├─ arrow-cpp 11.0.0 [required: 11.0.0, h9795d3c_1_cpu]
        │  └─ parquet-cpp 1.5.1 [required: >=0.11.0]
        │     └─ pyarrow 11.0.0 [required: 1.5.1.*]
        └─ pyarrow 11.0.0 [required: 11.0.0, h9795d3c_1_cpu]
```